clients/admin-ui/src/features/consent-reporting/hooks/useConsentReportingTableColumns.tsx

```diff
@@ -132,6 +132,11 @@ const useConsentReportingTableColumns = ({
         ),
         size: 120,
       }),
+      columnHelper.accessor((row) => row.url_recorded, {
+        id: "url_recorded",
+        cell: ({ getValue }) => <DefaultCell value={getValue()} />,
+        header: (props) => <DefaultCell value="Recorded URL" {...props} />,
```

Comment:
Somehow my review on this got removed. New attempt:
This needs to use the DefaultHeaderCell, that's why the font weight is off here
```suggestion
        header: (props) => <DefaultHeaderCell value="Recorded URL" {...props} />,
```
<img width="477" alt="image" src="https://github.com/user-attachments/assets/d70b7c5d-70cd-4c48-a2d6-0caa3f1f6d71" />